examples/igbh/dist_train_rgnn.py

```diff
@@ -187,14 +187,14 @@ def run_training_proc(local_proc_rank, num_nodes, node_rank, num_training_procs,
 
   best_accuracy = 0
   training_start = time.time()
-  for epoch in tqdm.tqdm(range(epochs)):
+  for epoch in range(epochs):
     model.train()
     total_loss = 0
     train_acc = 0
     idx = 0
     gpu_mem_alloc = 0
     epoch_start = time.time()
-    for batch in train_loader:
+    for batch in tqdm.tqdm(train_loader):
```

Comment:
Revert this change? As we need to use `tqdm` to create a progress bar for epochs.

Reply:
Epoch information is printed after validation. When an epoch costs much time, it can become hard to tell whether it gets stuck or is still running, so `tqdm` is moved down here.

Reply:
I've set the default `log_every` to 2 (can also be 1 if you think it's important) so that epoch info can be printed more frequently